packages/@aws-cdk/aws-cloudwatch/lib/metric.ts

```diff
@@ -823,6 +887,70 @@ export interface CreateAlarmOptions {
   readonly datapointsToAlarm?: number;
 }
 
+/**
+ * Properties needed to make an alarm from a metric
+ */
+export interface CreateAnomalyDetectionAlarmOptions {
```

Comment:
This should live in `anomaly-detection-alarm.ts` and should be renamed `AnomalyDetectionAlarmOptions`

Reply:
Done